docs/pages/_app.tsx

```diff
@@ -40,6 +41,7 @@ export default function Nextra({ Component, pageProps }: NextraAppProps) {
         </svg>
       </>
       <Component {...pageProps} />
+      <Analytics />
```

Comment:
this only seems to be inject client side. Not sure how to make it render in SSR so we get page views regardless. @shuding would this be a Nextra thing?